### ISSUE TITLE
outdated spf13/cobra dep 

### ISSUE THREAD

**Issue by andreykurilin** [kubernetes/kubernetes#29852]
Current dependency: Sun Apr 3 (4c05eb1145f16d0e6bb4a3e1b6d769f4713cb41f)

List of interesting fixed issues/prs in master of cobra:
- [Command with only hidden flags still shows "Flags:" section](https://github.com/spf13/cobra/issues/267).
- [Do not display bash commands if local flag specified](https://github.com/spf13/cobra/pull/288)
- [Sort commands by their names](https://github.com/spf13/cobra/pull/293)
- [Fix man page gen bugs](https://github.com/spf13/cobra/pull/294)
- [Help command must use Stdout instead of Stderr](https://github.com/spf13/cobra/pull/303)

**Comment by pwittrock** [comment]
@AdoHe Would you work with @andreykurilin on this?

**Comment by andreykurilin** [comment]
@pwittrock Does latest revision of PR requires additional work?) tests are green now...

**Comment by pwittrock** [comment]
@andreykurilin Have you filed a PR against the main repo?  I just see a commit against your repo.  You will need to click the "Create a PR" (or something like that) button from your repo to get the PR created.  Once you do that please reference it here using `#PR_NUMBER`

**Comment by pwittrock** [comment]
Also https://github.com/kubernetes/kubernetes/blob/master/docs/devel/development.md

**Comment by AdoHe** [comment]
@pwittrock I could help with this issue. @andreykurilin yes, please follow development doc to fill a PR.

**Comment by andreykurilin** [comment]
@pwittrock @AdoHe hm... I don't now why, but github did not connect my PR to this issue:( 
you can find it [here](https://github.com/kubernetes/kubernetes/pull/27855)

**Comment by AdoHe** [comment]
It's ok, I just see it.

**Comment by pwittrock** [comment]
@andreykurilin I've updated your comment to use the preferred style for referring in a PR.

**Comment by pwittrock** [comment]
@andreykurilin Ah I found your PR in the log.  The logs can be hard to parse and find stuff in, so just posting the PR number as a comment is sometimes the easiest way to help people find it.

**Comment by andreykurilin** [comment]
https://github.com/kubernetes/kubernetes/pull/27855 is merged
